wa/commands/create.py

```diff
@@ -35,6 +40,138 @@
+        self.parser.add_argument('-x', '--schema-version', action='store_true',
+                                 help='Display the current schema version.')
+
+    def execute(self, state, args):
+        self.get_schema(self.schemafilepath)
+        # Display the version if needed and exit
+        if args.schema_version:
+            self.logger.info(
+                'The current version is {}'.format(self.schemaversion))
+            sys.exit()
+        if not args.password:
+            self.parser.error(
+                'Please supply a password for the account used to connect.')
+        # Create the database (override if exists)
+        if args.dbname == 'postgres':
+            raise ValueError('Databasename to create cannot be postgres.')
+        database_exists = False
+        try:
+            test_conn = connect(dbname=args.dbname, user=args.username,
+                                password=args.password, host=args.host, port=args.port)
+            test_conn.close()
+            database_exists = True
+        except OperationalError:
+            pass
+
+        if database_exists and not args.force:
+            self.logger.error(
+                "Database {} already exists.".format(args.dbname) +
+                "Please specify the -f flag to create it afresh."
+            )
+            sys.exit()
+
+        conn = connect(dbname='postgres', user=args.username,
+                       password=args.password, host=args.host, port=args.port)
+
+        conn.set_isolation_level(ISOLATION_LEVEL_AUTOCOMMIT)
+        cursor = conn.cursor()
+        cursor.execute('DROP DATABASE IF EXISTS ' + args.dbname)
+        cursor.execute('CREATE DATABASE ' + args.dbname)
+        conn.commit()
+        cursor.close()
+        conn.close()
+
+        conn = connect(dbname=args.dbname, user=args.username,
+                       password=args.password, host=args.host, port=args.port)
+        cursor = conn.cursor()
+        cursor.execute(self.sqlstandardcommands)
+
+        extras.register_uuid()
+        cursor.execute("INSERT INTO DatabaseMeta VALUES (%s, %s, %s)",
+                       (
+                           uuid.uuid4(),
+                           self.schema_major,
+                           self.schema_minor
+                       )
+                       )
+
+        conn.commit()
+        cursor.close()
+        conn.close()
+
+        self.logger.info(
+            "Successfully created the database {}".format(args.dbname))
+
+        # Update config file
+        config_file = open(args.config_file, 'r')
+        config = yaml.load(config_file)
+        config_file.close()
+        if 'postgres' in config:
+            if not args.force_update_config:
+                self.logger.warning("The entry postgres already exists in the config file. " +
+                                    "Please specify the -m parameter to force an update.")
```

Comment:
Because the default is not to overwrite the config unless explicitly specified. This is for the user's benefit, to notify them of the fact that they can rerun the command with that flag to force update the config file too.